### ISSUE TITLE
Need option to disable checking of updates

### ISSUE THREAD

**Issue by ThomasPCG** [keybase/client#11237]
I use keybase once in a while so the frequency of updates annoy me. If there is a way to turn off update till the next time I decide to use the application or a notification to update the application when I start the application, it would be much better.

**Comment by 5h4d0ww0lf** [comment]
@ThomasPCG No, it won't be better you need frequent checks because there can be critical security update or incompatibility with the server if you use an outdated version.

**Comment by ThomasPCG** [comment]
Then could we at least have an option to only check for updates when I start the application? I only use the application occasionally and the updates pops up even when I did not start the application.

**Comment by cjb** [comment]
@ThomasPCG Out of curiosity, why not enable auto-updates to avoid being bothered by the prompt?

**Comment by ThomasPCG** [comment]
@cjb Just a personal belief that nothing should go on in my laptop without me knowing.. =D

**Comment by merwok** [comment]
It could become very messy if every app was trying to handle its own updates!
We have centralized package managers to make the experience nicer and give control to users.

**Comment by Sinjai** [comment]
I had to rename keybase.exe just so it would stop automatically restarting itself to check for updates constantly. Ridiculous.
